src/core/server/opensearch/version_check/ensure_opensearch_version.ts

```diff
@@ -35,7 +35,6 @@
 
 import { timer, of, from, Observable } from 'rxjs';
 import { map, distinctUntilChanged, catchError, exhaustMap, mergeMap } from 'rxjs/operators';
-import { get } from 'lodash';
 import { ApiResponse } from '@opensearch-project/opensearch';
```

Comment:
did you miss some changes for opensearch_service.ts?

```
            src/core/server/opensearch/opensearch_service.ts:98:7 - error TS2345: Argument of type '{ internalClient: Client; optimizedHealthcheckId: string | undefined; log: Logger; ignoreVersionMismatch: boolean; opensearchVersionCheckInterval: number; opensearchDashboardsVersion: string; }' is not assignable to parameter of type 'PollOpenSearchNodesVersionOptions'.
              Object literal may only specify known properties, but 'optimizedHealthcheckId' does not exist in type 'PollOpenSearchNodesVersionOptions'. Did you mean to write 'optimizedHealthcheck'?

            98       optimizedHealthcheckId: config.optimizedHealthcheckId,
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
never mind, it seems like that https://github.com/opensearch-project/OpenSearch-Dashboards/commit/e6bbb405441c7d59a8d5df6a477c5e432fa88b87 was not backport to 1.2 yet.